ts_src/payments/taprootutils.ts

```diff
@@ -46,94 +49,63 @@ export interface HashTree {
  *  - one taproot leaf and a list of elements
  */
```

Comment:
nit pick: js doc update
```suggestion
 *  - a taproot leaf [(output, version)], or
 *  - a taproot leaf (output, version), or
```

Reply:
I'll make a larger doc update, moving the description of the type to where it is defined, adding one for the HashTree type. Thanks!